components/org.wso2.carbon.identity.oauth.ciba/src/main/java/org/wso2/carbon/identity/oauth/ciba/exceptions/CibaClientException.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.oauth.ciba.exceptions;
+
+import org.wso2.carbon.identity.base.IdentityException;
+
+/**
+ * Client Exception.
+ */
+public class CibaClientException extends IdentityException {
+
+    private int status = 400;
```

Comment:
Do we need the status anywhere?

Reply:
yes status is essential at endpoints. Just added a default value which will anyways be overridden